tests/rules/property.js

```diff
@@ -111,7 +111,19 @@ eslintTester.run("property", rule, {
         {
             code: "w.innerHTML = `<span>${'lulz'+'meh'}</span>`;",
             parserOptions: { ecmaVersion: 6 }
-        }
+        },
+
+        // testing unwrapSafeHTML spread
+        {
+            code: "this.imeList.innerHTML = Sanitizer.unwrapSafeHTML(...listHtml);",
+            parserOptions: { ecmaVersion: 6 }
+        },
+
+
+        // Native method (Check customize code doesn't include these)
+        {
+            code: "document.toString = evil;"
```

Comment:
Prevents the bug I just asked you to close... I messed up on my own code, it's easy to do when doing the customisation objects.